platform/vendor/vendor.yaml

```diff
@@ -27,9 +27,9 @@ files:
     destination_dir: "tekton/catalog/main/task/kaniko/0.6"
     sha256: "595746871b33eb6dc84b56b1a94365d8707cbe7fcec91fb02218be834dce504c"
     validation_type: "sha256"
-  - release_file: "https://raw.githubusercontent.com/buildpacks/tekton-integration/main/task/buildpacks/0.4/buildpacks.yaml"
-    destination_dir: "buildpacks/tekton-integration/main/task/buildpacks/0.4"
-    sha256: "9e48d561fd08bf3d434cde025b45b57adca816c930d1cc653a8592a33f394513"
+  - release_file: "https://raw.githubusercontent.com/tektoncd/catalog/main/task/buildpacks/0.5/buildpacks.yaml"
+    destination_dir: "tekton/catalog/main/task/buildpacks/0.4"
```

Comment:
Should the destination be updated from 0.4 -> 0.5? It means we need to change all the code pointing to the old version.

Reply:
Yep, silly mistake on my part. 😊